collect_env.py

```diff
@@ -512,6 +515,19 @@ def is_xnnpack_available():
     else:
         return "N/A"
 
+def get_env_vars():
+    env_vars = ''
+    secret_terms=('secret', 'token', 'api', 'access', 'password')
+    report_prefix = ("TORCH", "NCCL")
```

Comment:
some torch env vars start with `PYTORCH`

```suggestion
    report_prefix = ("TORCH", "NCCL", "PYTORCH")
```

Reply:
and actually, from https://pytorch.org/docs/stable/cuda_environment_variables.html

* `CUDA`
* `CUBLAS`
* `CUDNN`
* `NVIDIA`

more here https://pytorch.org/docs/stable/threading_environment_variables.html ...

not sure how far to go here ...

Reply:
Done. If there are some important env variables missing in the future, it is easy to append more.